Uplift/Models/Gym.swift

```diff
@@ -135,136 +126,189 @@ struct DailyGymHours {
             openTime = Date.getTimeFromString(datetime: gymHoursData.startTime)
             closeTime = Date.getTimeFromString(datetime: gymHoursData.endTime)
         } else {
-            openTime = Date()
             closeTime = openTime
-            dayOfWeek = 0
         }
     }
 
-    init(facilityHoursData: AllGymsQuery.Data.Gym.Facility.Time?) {
-        if let facilityHours = facilityHoursData {
-            dayOfWeek = facilityHours.day
-            openTime = Date.getTimeFromString(datetime: facilityHours.startTime)
-            closeTime = Date.getTimeFromString(datetime: facilityHours.endTime)
-        } else {
-            openTime = Date()
-            closeTime = openTime
-            dayOfWeek = 0
-        }
+}
+
+// MARK: - Facilities
+
+struct DailyFacilityHoursRanges {
+    var dayOfWeek: Int
+    var timeRanges: [FacilityHoursRange] = []
```

Comment:
don't think we have to give it a default value right?

Reply:
you are right

Reply:
forgot to remove them when johnathan made the fields non nullable